src/schemathesis/models.py

```diff
@@ -463,6 +464,8 @@ def validate_response(
         from .checks import ALL_CHECKS
 
         checks = checks or ALL_CHECKS
+        checks = tuple(set(checks) - set(excluded_checks))
```

Comment:
I'd suggest using something like this to maintain a stable checks order which appears later on in failure reporting:

```python
checks = tuple(check for check in checks if check not in excluded_checks)
```

P.S. I assume it should be fine not to use sets as the container size is expected to be just a few elements

Reply:
Awesome suggestion, that makes it much easier to read as well, thanks! I've made those changes and amended that commit to include these changes (also made the same change in schemas.py:73